animation_nodes/nodes/action/chain_action_list.pyx

```diff
@@ -0,0 +1,131 @@
+import bpy
+from bpy.props import *
+from ... events import propertyChanged
+from ... base_types import AnimationNode
+from ... graphics import Rectangle
+from ... graphics.drawing_2d import drawVerticalLine
+from ... data_structures cimport (
+    BoundedAction, BoundedActionEvaluator,
+    FloatList, BooleanList
+)
+
+class ChainActionListNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_ChainActionListNode"
+    bl_label = "Chain Action List"
+    errorHandlingType = "EXCEPTION"
+
+    __annotations__ = {}
+    __annotations__["relative"] = BoolProperty(name = "Relative", default = True,
+        description = "Use last frame of first action as starting point of the second",
+        update = propertyChanged)
+
+    def create(self):
+        self.newInput("Action List", "Action list", "list")
```

Comment:
Use more explicit names, `list` -> `actions`, `r` -> `chainedAction`, and so on.